sql/tidb-specific.md

```diff
@@ -282,6 +282,13 @@ set @@global.tidb_distsql_scan_concurrency = 10
 - This variable is used to set the priority of executing the `ADD INDEX` operation in the `re-organize` phase.
 - You can set the value of this variable to `PRIORITY_LOW`, `PRIORITY_NORMAL` or `PRIORITY_HIGH`.
 
+### tidb_force_priority
+
+- Scope: SESSION | GLOBAL
```

Comment:
It is a session-only variable though it works on the server level. `GLOBAL` variable will work on all TiDB servers.